### ISSUE TITLE
[improvement] Add option to make a project public (from start)

### ISSUE THREAD

**Issue by combinatorist** [Swing-team/pendulums-web-client#45]
We track volunteer time at Code for Nashville (for fundraising, professional development, etc), so it would be nice to point the public to a page that shows a live feed of the time volunteers are spending on various projects.

We could do that with Toggl, but I'd rather use something open source and I love that you have unlimited teams. But, as far as I can tell, there's no way to make a project public on your website. That seems like it would fairly easy to do, just letting project owners turn off the permissions barrier. You could even take a slack approach and insist that any project that started private is not allowed to become public or at if it is converted, then it's private history stays private (more complicated).

We would be happy to help implement this if it is something that seems valuable / you would want on your roadmap.

**Comment by mohammadrafigh** [comment]
Thank you @combinatorist for reminding this. this feature is already listed in our backlog from Oct 2019. we scheduled a release for next 2, 3 weeks so I move this feature up in our backlog and you can expect this feature for next release. Thanks for your interest on helping us but IMO the client code needs some refactor and documentation to be easier for others to contribute. If we were not able to put this feature in our next release, I will prepare the API for you and ask your contribution to implement this in client (if you were still available to help us). thanks again for being interested in Pendulums.

**Comment by combinatorist** [comment]
Awesome - thanks so much, @mohammadrafigh!

**Comment by combinatorist** [comment]
It seems this item is in progress, but there hasn't been any release
recently. Am interpreting that correctly or should I look at a different t
repo for the release?

**Comment by mohammadrafigh** [comment]
Yes it's listed in our current sprint. It takes some time because to
provide this feature we needed to redesign some components and we are at
the final stage of this change, after that the public projects could be
implemented easily.
We will share our plans soon in an appropriate way. But for now if you are
curious about the changes you can track develop and feature branches.

**Comment by combinatorist** [comment]
Awesome - thank you!

**Comment by mohammadrafigh** [comment]
To update you and anyone interested here, We are currently implementing the new project page which could be exposed publicly using a link (later we will include a search through public projects from landing page). Sorry if it takes more time because we are a small team without any support from community so we have a lot to do...

**Comment by combinatorist** [comment]
Great - thank you! So actually, how could community members like me help?

**Comment by mohammadrafigh** [comment]
Thank you for being patient. The community could help the project with
donations. We have a link both in app and landing page. We support
cryptocurrency payments and giftcards. Currently the company is deciding to
open source the backend too if we get enough donations.

**Comment by combinatorist** [comment]
Got it - that makes sense. How much is enough donations to open source the
backend? I'm part of a national network who might be willing to donate and
contribute code, if we understood the terms.

**Comment by mohammadrafigh** [comment]
Thanks for being interested. Can you please contact us using [this link](https://pendulums.io/contact-us.html) so we can stay connected and talk about donations easier?

**Comment by combinatorist** [comment]
Done. Thanks, @mohammadrafigh!
